clarity/src/vm/docs/mod.rs

```diff
@@ -529,7 +529,7 @@ const LOG2_API: SimpleFunctionAPI = SimpleFunctionAPI {
     snippet: "log2 ${1:expr-1}",
     signature: "(log2 n)",
     description:
-        "Returns the power to which the number 2 must be raised to to obtain the value `n`, rounded 
+        "Returns the power to which the number 2 must be raised to obtain the value `n`, rounded 
```

Comment:
This one was correct as "to to". It is the two phrases "raised to" and "to obtain".

Reply:
Ah, I'm sorry. I just re-read this and realized it was correct with just one "to".